### ISSUE TITLE
Only TestFlight bug - Image Not Found

### ISSUE THREAD

**Issue by AdlibBeats** [realm/realm-js#4140]
### How frequently does the bug occur?

All the time

### Description

Crash only TestFlight on iOS ... < 15.0

### Stacktrace & log output

```shell
{"app_name":"test","app_cohort":"2|date=1639096200000&sf=143469&tid=ce768aa66a3d199a9fca7ebbc6d5301ffc41fd4426e910468d2e396485202005&ttype=i","app_version":"1.1.6","timestamp":"2021-12-10 03:45:14.54 +0300","slice_uuid":"aa178c58-89ec-38b1-905b-4134f9770992","adam_id":1533641502,"build_version":"108","bundleID":"xxx.test","share_with_app_devs":false,"is_first_party":false,"bug_type":"109","os_version":"iPhone OS 12.5.5 (16H62)","incident_id":"6C234363-78C3-4962-A52D-0F7C276F8E40","is_beta":true,"name":"test"}
Incident Identifier: 6C234363-78C3-4962-A52D-0F7C276F8E40
Beta Identifier:     7E045794-93A6-4129-84B4-2C8A5D9EA5FF
Hardware Model:      iPhone6,2
Process:             test [147]
Path:                /private/var/containers/Bundle/Application/7EECE82C-35E1-42B1-B8E3-710F040DD726/test.app/test
Identifier:          xxx.test
Version:             108 (1.1.6)
AppStoreTools:       13C88a
AppVariant:          1:iPhone6,2:12.2
Beta:                YES
Code Type:           ARM-64 (Native)
Role:                Foreground
Parent Process:      launchd [1]
Coalition:           xxx.test [248]


Date/Time:           2021-12-10 03:45:11.8437 +0300
Launch Time:         2021-12-10 03:45:04.7125 +0300
OS Version:          iPhone OS 12.5.5 (16H62)
Baseband Version:    10.80.02
Report Version:      104

Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Description: DYLD, Library not loaded: @rpath/libswift_Concurrency.dylib | Referenced from: /private/var/containers/Bundle/Application/7EECE82C-35E1-42B1-B8E3-710F040DD726/test.app/Frameworks/RealmSwift.framework/RealmSwift | Reason: image not found
Triggered by Thread:  0
```

### Can you reproduce the bug?

Yes, always

### Reproduction Steps

Just build any application, -> TestFlight (Any iPhone with iOS ... < 15.0)

### Version

10.19.0

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

... < 15.0

### Build environment

Which debugger for React Native: ..

**Comment by kneth** [comment]
@AdlibBeats I wonder if you have created the issue in the wrong repository. Realm JS hasn't released 10.19.0 yet while [Realm Cocoa](https://github.com/realm/realm-cocoa) has. Moreover, the error message `Library not loaded: @rpath/libswift_Concurrency.dylib` indicates that you are using Swift.

**Comment by bmunkholm** [comment]
@AdlibBeats Closing this issue. Please follow https://github.com/realm/realm-swift/issues/7573 instead, Thanks!
